.github/workflows/build_qgis_plugin.yml

```diff
@@ -35,10 +35,12 @@ jobs:
     - name: Prepare artifact
       run: |
         rm bin/qgis/smoderp2d-plugin/zip_build/*.zip
+        ls -a bin/qgis/smoderp2d-plugin/zip_build/smoderp2d_plugin/smoderp2d
```

Comment:
Just curious - why is this `ls -a` needed here?

Reply:
Good catch. It's a debug statement, which hasn't been removed. Done by ec3b4b18bcc42edd1ddb766f32ad802c279227c8

Reply:
I thought so. Cool, thanks. Resolving the discussion.